test/runTest.sh

```diff
@@ -1,16 +1,32 @@
 #!/bin/bash
 
+export PGDATESTYLE="Postgres, MDY";
+psql -c "SHOW DateStyle;"
+
 createdb temporal_tables_test
 psql temporal_tables_test -q -f versioning_function.sql
 psql temporal_tables_test -q -f system_time_function.sql
 
 mkdir -p test/result
 
+FILES_DIFFERENT=false
+
+REMOTE_TESTS="
+  combinations structure versioning
+  invalid_system_period invalid_system_period_values invalid_types
+  no_history_system_period no_history_table no_system_period
```

Comment:
The "test/runTestNochecks.sh" originally did not include the below 6 tests.
```
invalid_system_period
invalid_system_period_values
invalid_types
no_history_system_period
no_history_table
no_system_period
```
I am not sure about the original reason. But it makes sense to add now.

Reply:
I believe that unless we can think of a reason why they aren't included, we can assume they may have just been forgotten. Let's try adding them and see what happens

Reply:
I tried adding these test cases to "no checks" file. All of them are failing for some reason. Reviewing them now.

Reply:
ok then maybe there is a reason. let's just confirm that the reason is valid, then we decide what to do

Reply:
I think, I found the reason behind ignoring these tests. All the above tests are unhappy paths.
The versioning function defined in `versioning_function_nochecks.sql` does not sanitize things to prevent failure unlike `versioning_function.sql`.
That's the idea behind the "no checks" file I believe. Run it faster and ignore any checks and sanitization.
They have captured it here in the README.md file as well - https://github.com/nearform/temporal_tables?tab=readme-ov-file#test

> Obviously, this suite won't run the tests about the error reporting.

Reply:
thanks for checking. we're good then 👌